Polyfills/Window/Source/Window.cpp

```diff
@@ -53,6 +58,18 @@ namespace Babylon::Polyfills::Internal
         {
             global.Set(JS_REMOVE_EVENT_LISTENER_NAME, Napi::Function::New(env, &Window::RemoveEventListener, JS_REMOVE_EVENT_LISTENER_NAME));
         }
+        
+        //global.Set(JS_DEVICE_PIXEL_RATIO_NAME,Napi::PropertyDescriptor::Accessor(env, global, JS_DEVICE_PIXEL_RATIO_NAME, &Window::GetDevicePixelRatio, napi_default, nullptr));
+        //auto prop = Napi::PropertyDescriptor::Accessor(env, global, JS_DEVICE_PIXEL_RATIO_NAME, &Window::GetDevicePixelRatio, napi_default, nullptr);
+        //global.DefineProperty(prop);
+
+        // Create an accessor to add to the window object to define window.devicePixelRatio
+        Napi::Object descriptor{Napi::Object::New(env)};
+        descriptor.Set("enumerable", Napi::Value::From(env, true));
+        descriptor.Set("get", Napi::Function::New(env, &Window::GetDevicePixelRatio, "devicePixelRatio", jsWindow));
+        Napi::Object object{global.Get("Object").As<Napi::Object>()};
+        Napi::Function defineProperty{object.Get("defineProperty").As<Napi::Function>()};
+        defineProperty.Call(object, {global, Napi::String::New(env, "devicePixelRatio"), descriptor});
```

Comment:
It would probably be good to wrap this with an if check like the other global members in case this property already exists.

Reply:
Ok, makes sense. I'll clean up this portion.